tests/test_files/calculator/calculator.py

```diff
@@ -1,5 +1,24 @@
-# TODO: Implement a calculator program, that can add, subtract, multiply and divide two numbers.
+# Implement a calculator program, that can add, subtract, multiply and divide two numbers.
 #  - Numbers and operators given from the user as the command line arguments.
 #  - Consider using argpasre library.
 #  - Consider using eval builtin function.
 #  - Print or log the result.
+import argparse
+
+def check_number(value):
+    try:
+        float(value)
+        return value
+    except ValueError:
+        raise argparse.ArgumentTypeError("Argument is not a number")
+
+parser = argparse.ArgumentParser()
+parser.add_argument("number1", type=check_number, help="First number")
+parser.add_argument("number2", type=check_number, help="Second number")
+parser.add_argument("operation", choices=["+", "-", "*", "/"], help="Operation")
```

Comment:
As usual arguments written as: --operation or -o (as flag). Please try use the same pattern, but not everywhere because we want to test what will do the argument parser in that case.
Fun fact --help and -h arguments are reserved, if you want try that out.
This whole comment applies to the most of programming language.